generator/uscore/uscore_generator.rb

```diff
@@ -1450,9 +1450,9 @@ def search_param_match_found_code(type, element)
             coding_value = value.split('|').last
             match_found = values_found.any? do |codeable_concept|
               if value.include? '|'
-                codeable_concept.coding.any? { |coding| coding.system == coding_system && coding.code == coding_value }
+                codeable_concept.coding.any? { |coding| coding.system == coding_system && coding.code&.casecmp?(coding_value) }
               else
-                codeable_concept.coding.any? { |coding| coding.code == value }
+                codeable_concept.coding.any? { |coding| coding.code&.casecmp?(value) }
```

Comment:
Looks good.  I was a little concerned about the behavior if `coding.code` is `nil` here, but it works out... if `coding.code` is `nil` then the block will return `nil` which is evaluated as `false`, which is what we want.

Reply:
I also check what happens if `value` is nil, and if that is the case, then this will also return nil, which is evaluated as false, which works out.  Bottom line, this is fine.  I just wanted to note that I checked those two things :)